crates/astria-conductor/src/executor.rs

```diff
@@ -114,6 +118,12 @@ struct Executor<C> {
     /// so that we can mark the block as final on the execution layer when
     /// we receive a finalized sequencer block.
     sequencer_hash_to_execution_hash: HashMap<Vec<u8>, Vec<u8>>,
+
+    /// most recently executed sequencer block hash
+    most_recently_executed_block_hash: Hash,
+
+    /// block queue for blocks that have been recieved but their parent has not been executed yet
+    block_queue: PriorityQueue<SequencerBlockData, u64>,
```

Comment:
is the `u64` here the priority?

Reply:
also how does the priority work? is the lowest number popped first or no?

Reply:
Yup. I'm just using the block height for this.

Reply:
and yes the lowest block in the queue will get popped first.
see the `try_execute_queue` function. the queue sorts highest to lowest so i'm actually popping the last item